drivers/performance_measurement/app/models/performance_measurement/report.rb

```diff
@@ -1104,7 +1108,7 @@ def spm_fields
           ],
         },
         {
-          cells: [['1a', 'B2']],
+          cells: [['1a', 'D2']],
```

Comment:
I believe these are incorrect. However it doesn't seem to matter in practice as cell is just used to get the universe from the SPM, and all all cells have the same universe